ember-stargate/package.json

```diff
@@ -38,7 +38,7 @@
     "@ember/render-modifiers": "^2.1.0",
     "@embroider/addon-shim": "^1.8.9",
     "@glimmer/component": "^1.1.2",
-    "ember-resources": "^7.0.1",
+    "ember-resources": "^7.0.3",
```

Comment:
The change is fine, but it wasn't really blocking anyone, given that the caret version range would have allowed to use the latest. Just a matter of updating your lockfile, right?